src/audio/codec_adapter/codec/cadence.c

```diff
@@ -90,7 +90,7 @@ int cadence_codec_init(struct comp_dev *dev)
 
 	comp_dbg(dev, "cadence_codec_init() start");
 
-	cd = codec_allocate_memory(dev, sizeof(struct cadence_codec_data), 0);
+	cd = comp_devm_alloc(dev, sizeof(struct cadence_codec_data), 0);
 	if (!cd) {
```

Comment:
I wonder if this shouldn't be better called devm_comp_alloc(dev, ..) similar with its Linux counterpart.

Reply:
yes sure. Can revisit the naming as I need to shuffle this around anyway...